src/visualization/types/SimpleTable/index.tsx

```diff
@@ -33,6 +33,7 @@ export default register => {
     graphic: icon,
     initial: {type: 'simple-table', showAll: false},
     component: view,
+    featureFlag: 'useGiraffeGraphs',
```

Comment:
what is your understanding of this property?

Reply:
It seems like a convenient way to put the featureFlag in one place and then use it for that graph, i noticed the UI access those properties when deciding what graph types to expose in the UI, through the dropdown.